packages/angular-material/test/horizontal-layout.spec.ts

```diff
@@ -45,7 +44,7 @@ describe('Horizontal layout', () => {
   beforeEach(() => {
     fixture = beforeEachLayoutTest(HorizontalLayoutRenderer, {
       declarations: [LayoutChildrenRenderPropsPipe],
-      imports: [FlexLayoutModule],
+      imports: [],
```

Comment:
can this be removed now that it's empty?

Reply:
sure